<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>94</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/search</normaloff>:/icons/search</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Search...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/clear</normaloff>:/icons/clear</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="originalBox">
       <property name="text">
        <string>Original</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="translatedBox">
       <property name="text">
        <string>Translated</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="commentBox">
       <property name="text">
        <string>Comment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="caseBox">
       <property name="text">
        <string>Case Sensitive</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>searchEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>searchEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commentBox</sender>
   <signal>clicked()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>originalBox</sender>
   <signal>clicked()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>translatedBox</sender>
   <signal>clicked()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>caseBox</sender>
   <signal>clicked()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
